ocs_ci/ocs/platform_nodes.py

```diff
@@ -79,6 +79,7 @@ def __init__(self):
             "rhv": RHVNodes,
             "ibm_cloud": IBMCloud,
             "vsphere_ipi": VMWareIPINodes,
+            "rosa": AWSNodes,
```

Comment:
As ROSANodes class is defined, it should be added here. That said, I think that not having ROSANodes and just using AWSNodes works too.

Reply:
I think we can add the `ROSANodes` class here because in the future we might want to have functions related to rosa.